src/components/InvoiceList/index.jsx

```diff
@@ -4,22 +4,39 @@ import InvoiceSummary from "../InvoiceSummary"
 export default function InvoiceList() {
 
     const [invoices, setInvoices] = useState([])
+    const [filterInvoices, setFilterInvoices] = useState(NaN)
```

Comment:
Intentionally using NaN is pretty wild. Just use 0 instead since that's never a valid status

Reply:
If i make this 0, it doesn't work. becauase status 0 doesn't exist, so when you first load the page there are no invoices.

Reply:
Chuck an if statement into your useEffect to only apply the filter if the filter is not 0. Sending some strange value to an API (NaN) has it's own risks as well, in this case it works but in others it may not. It would be ideal to not have the ?status at all if no filter has been set.

The best way to do it is to make a variable before the if statement that has the normal unfiltered URL, and then add the ?status onto it if filter != 0, and then pass that variable into the fetch request